### DISCUSSION TITLE
Why is "anonymous" when i use rest api to create a session?

### DISCUSSION THREAD

**Discussion by BruceWong96** [apache/kyuubi#5013] [Q&A] [answered]
<img width="1375" alt="image" src="https://github.com/apache/kyuubi/assets/29974394/b925765f-dcfc-4e26-b33f-e8cba0a78f8e">
<img width="1083" alt="image" src="https://github.com/apache/kyuubi/assets/29974394/f4bc2eb7-57ab-4744-a900-81f9e135e0bd">

So, what should i do ?
Thank u.

**Comment by HaoYang670** [comment, marked as answer]
This is because you don't provide a username. Please try to set an authentication header in the REST request.

**Reply by BruceWong96** [reply]
Thanks
